Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.h

```diff
@@ -29,15 +32,26 @@ class SCENE_SURFACE_MESH_ITEM_EXPORT Scene_surface_mesh_item
   typedef CGAL::Exact_predicates_inexact_constructions_kernel Kernel;
   typedef Kernel::Point_3 Point;
   typedef CGAL::Surface_mesh<Point> SMesh;
+  typedef SMesh FaceGraph;
   typedef boost::graph_traits<SMesh>::face_descriptor face_descriptor;
+  typedef boost::graph_traits<SMesh>::vertex_descriptor vertex_descriptor;
+  typedef boost::graph_traits<SMesh>::halfedge_descriptor halfedge_descriptor;
+  typedef SMesh::Property_map<vertex_descriptor,int> Vertex_selection_map;
+  typedef SMesh::Property_map<face_descriptor,int> Face_selection_map;
+  typedef SMesh::Property_map<halfedge_descriptor, bool> Halfedge_is_feature_map;
+  typedef SMesh::Property_map<face_descriptor, int> Face_patch_id_map;
+  typedef SMesh::Property_map<vertex_descriptor,int> Vertex_num_feature_edges_map;
```

Comment:
No `Surface_mesh_type.h` may not be always included. For a plugin with a lot of code, like the mesh_3_plugin, the part of the plugin that actually deal with CGAL code is in a dedicated .cpp files, and the main .cpp of the plugin only includes `Scene_polyhedron_item.h`, and not `Polyhedron_type.h`. That way, the main .cpp file of the plugin, that deals with the Qt user interface of the plugin, does not include the CGAL headers, but only Qt5 headers.